app/helper/rtUploadAttachment.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require("../../../../../wp-load.php");
```

Comment:
Instead of loading `wp-load.php` like this, can we load it like this?
```
$parse_uri = explode( 'wp-content', $_SERVER['SCRIPT_FILENAME'] );
require_once( $parse_uri[0] . 'wp-load.php' );
```